docs/Installation.md

```diff
@@ -40,6 +40,8 @@ If you are a Windows user who is new to Python and TensorFlow, follow [this guid
 
 ### Mac and Unix Users
 
+If you do not have Python, you can download and install Python 3 from [here](https://www.python.org/downloads/).
```

Comment:
I'd slightly re-word to be more inline with our other doc: [Download](https://www.python.org/downloads/) and install Python 3 if you do not already have it.

Reply:
done